### PR TITLE
feat: add webui intro

### PR DESCRIPTION
**Pull request by fsdiogo**
See https://github.com/ipfs-shipyard/ipfs-webui/pull/847#issuecomment-427060102.

### CODE REVIEW

**Comment by fsdiogo** [comment]
First pass at this:

## When there are no files

welcome

## When using Companion

companion

@olizilla @lidel @hacdias what do you think? The copy needs some tweaks too.

**Comment by fsdiogo** [comment]
📸 	incoming:

## Companion `with` files

companion-files

## Companion `without` files

companion-no-files

## Daemon `with` files

files

## Daemon `without` files

no-files

**Comment by fsdiogo** [comment]
Going to add info about drag and drop as noted in https://github.com/ipfs-shipyard/ipfs-webui/issues/818#issuecomment-426974387.

**Comment by olizilla** [comment]
Moved the welcome messages above the files list, made the add files button look like the main thing, so bigger and and green, to line up with the size column below, and tweaked the copy.

Update the logic to only show messages when we're at `/`. Once a the user starts navigating around, we can assume they have got the idea that window.ipfs is hiding things.

### Using API at root no files

screenshot 2018-10-05 10 44 33

### Using API with some files

screenshot 2018-10-05 10 47 15

### Using window.ipfs at root and no files

screenshot 2018-10-05 10 50 26

### Using window.ipfs at root with files

screenshot 2018-10-05 10 50 41

### Using window.ipfs not at root

screenshot 2018-10-05 10 53 30

**Comment by olizilla** [comment]
We also need a bunch more i18next work on this and some of the files list and the welcome page.

**Comment by hacdias** [comment]
I agree with all @olizilla has said about the placement of the boxes: I think it's better to put them below the buttons and the headers.

Also, @lidel made a good point referring to the looks of the 'Explore' button. Perhaps it would be better to have the default background colour set to the muted version and change it to the regular, darker colour on hover.

**Comment by olizilla** [comment]
@lidel yep, thats reasonable. I'm going to get this merged first, and will look at that later, as the explore form is in the ipld-explorer-components repo

**Comment by olizilla** [comment]
Ok, I've moved all the empty files text and welcome page text into locales for translating, and fixed some minor UI issues like #849 

@hacdias @fsdiogo can you check it all makes sense please.

**Comment by fsdiogo** [comment]
It LGTM @olizilla :+1:

Just one minor thing, I personally would remove the red portion as we already have the message below the breadcrumbs.

ss

**Comment by fsdiogo** [comment]
Good thinking 👍

File: public/locales/en/files.json [hunk 1]
```diff
@@ -58,5 +58,19 @@
   "newFolderModal": {
     "title": "New folder",
     "description": "Insert the name of the folder you want to create."
-  }
+  },
+  "welcomeInfo": {
+    "header": "Welcome to the IPFS Web UI",
+    "paragraph1": "<0><0>Check the status</0> of your node, it's Peer ID and connection info, the network traffic and the number of connected peers.</0>",
```
[the remark this answers is not in the document]
Comment (olizilla) [reply public/locales/en/files.json]: Yeah, that is what I thought to, but this is copy and pasted from the i8next console message. In practice it works, but it looks pretty confusing. I could try editing it and seeing if it still works...

**Pull request merged**